lightning/src/ln/channel.rs

```diff
@@ -4095,6 +4119,10 @@ impl<Signer: Sign> Channel<Signer> {
 			}
 		}
 
+		if outbound_dusted_htlc_value_msat + amount_msat > self.get_max_balance_dust_htlc_msat() {
```

Comment:
Shouldn't this be gated on `amount_msat` actually being below the dust limit as well?